src/attr/_make.py

```diff
@@ -1757,9 +1835,19 @@ def from_counting_attr(cls, name, ca, type=None):
             validator=ca._validator,
             default=ca._default,
             type=type,
+            cmp=None,
             **inst_dict
         )
 
+    @property
+    def cmp(self):
+        """
+        Simulate the presence of a cmp attribute and warn.
+        """
+        warnings.warn(_CMP_DEPRECATION, DeprecationWarning, stacklevel=2)
```

Comment:
Is this (accessing Attribute.cmp directly) tested somewhere?

(The deprecation warning, and whether it gives the right answer both if you used eq and / or cmp?)

Reply:
The deprecation is tested in https://github.com/python-attrs/attrs/pull/574/files#diff-11d337c4d38019bd30952e7e1ef21ec0R624-R637

I guess I [cs]ould add a test for the values too.